options/options_helper.h

```diff
@@ -204,9 +204,6 @@ static std::unordered_map<std::string, OptionTypeInfo> db_options_type_info = {
     {"compaction_readahead_size",
      {offsetof(struct DBOptions, compaction_readahead_size), OptionType::kSizeT,
       OptionVerificationType::kNormal, false, 0}},
-    {"random_access_max_buffer_size",
-     {offsetof(struct DBOptions, random_access_max_buffer_size),
-      OptionType::kSizeT, OptionVerificationType::kNormal, false, 0}},
```

Comment:
We need to leave it as OptionVerificationType::kDeprecated so that we can load DB generated by older versions.